relayer/relays/beefy/scanner.go

```diff
@@ -108,7 +109,7 @@ func scanBlocks(ctx context.Context, meta *types.Metadata, api *gsrpc.SubstrateA
 		}
 
 		if sessionIndex > currentSessionIndex {
-			currentSessionIndex = sessionIndex
+			//currentSessionIndex = sessionIndex
```

Comment:
Do not increment the session, which essentially makes it try every block in the next session, pushing all found commitments through instead of the first commitment per session.